packages/playwright-core/src/server/registry/nativeDeps.ts

```diff
@@ -642,7 +643,7 @@ export const deps: any = {
       'libicuuc.so.66': 'libicu70',
       'libicui18n.so.66': 'libicu70',
       'libwebp.so.6': 'libwebp6',
-      'libenchant.so.1': 'libenchant-2-2',
+      'libenchant-2.so.2': 'libenchant-2-2',
```

Comment:
Should we keep both of them? I don't think there is libenchant-2.so.2 on Ubuntu 20 or is there?

Reply:
Oh, nm, it's 22.